src/System.Runtime.Intrinsics/src/System/Runtime/Intrinsics/Types.cs

```diff
@@ -0,0 +1,18 @@
+using System.Runtime.InteropServices;
+
+namespace System.Runtime.Intrinsics
+{
+    // 128 bit types
+    [StructLayout(LayoutKind.Sequential, Size = 16)]
+    public struct Vector128<T> where T : struct 
```

Comment:
I think the convention is generally one type per file, but it isn't explicitly documented in the coding style guidelines. 

Maybe someone else can comment?